9.tcga-classify/README.md

```diff
@@ -17,7 +17,7 @@ Observing how these compressed features participated in the models (by model coe
 
 **Notes**
 
-* Samples with a hypermutater phenotype were removed from the dataset (greater than 5 standard deviations over the mean of log10 mutation rate)
+* Samples with a hypermutater phenotype were removed from the data set (greater than 5 standard deviations over the mean of log10 mutation rate)
```

Comment:
Not sure if this is a typo, "with greater than 15% of what?"

Reply:
this comment is for line 24 I think. I will update in the next commit to clarify what I mean. Thanks!